pipdb.js

```diff
@@ -0,0 +1,114 @@
+require('buffertools').extend();
+
+var dbIndices = {};
+var dbTypes = {};
+
+var dbEntryType = ['Bool', 'Int8', 'UInt8', 'Int32', 'UInt32', 'Float', 'String', 'List', 'Dictionary'];
+
```

Comment:
Ok, always happy to `git cherry pick`.

Perhaps I should have outlined it in each. My intention was that bits that would be useful by any node application would go to `pipboylib` while `pipboyrelay` is one cli tool that uses them.